3.4/graphs.md

```diff
@@ -56,7 +56,7 @@ However the graph module adds an additional layer on top of these collections gi
 * If you delete a vertex all edges will be deleted, you will never have loose ends
```

Comment:
Maybe mention here that you *can* have loose ends if you start with a dataset which has dangling edges to begin with?

Reply:
```suggestion
* If you delete a vertex all edges will be deleted
```
Removing the last part and then adding something in the following paragraph about if the dataset starts with dangling edges they will not be removed upon creation seems like a good idea.